config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :greetings, only: [:index]
```

Comment:
## [OPTIONAL]

- [ ] It is always a good practice prefixing your based_url with `api` + the version example: 'http://mybackend_domain_name/api/v2'. This is very important you can go through this article to understand [why and how](https://medium.com/swlh/a-deeper-dive-into-api-versioning-938b0cb58765).